integration-tests/src/test/resources/bash-scripts/discover_domain.sh

```diff
@@ -0,0 +1,257 @@
+#!/bin/bash
```

Comment:
There is a similar script ( installer/src/main/bin/discoverDomain.sh)  available on the WDT tool itself.  Can we not use the same tool to generate the model file. If we mantain out own copy, we will be out-of-sync

Reply:
This script is not wdt discoverDomain tool. This is script that I wrote to run inside a pod with weblogic. This script downloads wdt and also invokes wdt discoverDomain tool to generate the files. Maybe a bad choice of filename, but I wanted the file name to reflect what that script is for. I am not copying any scripts from wdt or any other tool.

Reply:
Did you thought of changing the file name ?

Reply:
No, I do not want to change the name. There is no conflict. WDT  has discoverDomain.sh - not the same name